bubblegum/program/src/lib.rs

```diff
@@ -119,6 +126,42 @@ pub struct CreatorVerification<'info> {
     pub merkle_slab: UncheckedAccount<'info>,
 }
 
+#[derive(Accounts)]
+pub struct CollectionVerification<'info> {
+    #[account(
+        seeds = [merkle_slab.key().as_ref()],
+        bump,
+    )]
+    pub authority: Account<'info, TreeConfig>,
+    /// CHECK: This account is checked in the instruction
+    pub owner: UncheckedAccount<'info>,
+    /// CHECK: This account is checked in the instruction
+    pub delegate: UncheckedAccount<'info>,
+    pub payer: Signer<'info>,
+    /// CHECK: This account is checked to be a signer in
+    /// the case of `set_and_verify_collection` where
+    /// we are actually changing the NFT metadata.
+    pub tree_delegate: UncheckedAccount<'info>,
+    pub collection_authority: Signer<'info>,
+    /// CHECK: This account is checked in the instruction
+    pub collection_mint: UncheckedAccount<'info>,
+    pub collection_metadata: Box<Account<'info, TokenMetadata>>,
+    /// CHECK: This account is checked in the instruction
+    pub edition_account: UncheckedAccount<'info>,
+    /// CHECK: This is just used as a signing PDA.
+    #[account(
+        seeds = [COLLECTION_CPI_PREFIX.as_ref()],
```

Comment:
I just put a string here for a seed.  In actuality I don't think I need any seed, this is really only a signing PDA and the only thing that matters is that it is owned by Bubblegum and can be a signer for a CPI into token-metadata.

That being said, do any of you have any feedback or suggestions for different PDA seeds?  I could leave this as-is, or have `seeds = []`, or also include `merkle_slab.key()` like in most of the other Bubblegum PDAs.

Reply:
In this case, I feel that you only need a single seed, and that the seed should indicate what the PDA's purpose is. Basically, self-documentation. I think "signer" or "program_signer" makes sense but don't have a super strong opinion otherwise. If you think it's possible you'd want multiple program signers in the future, then you'd want to add a second seed to differentiate what kind of signer or what purpose it is for.

Reply:
I'm using "collection_cpi" for now, can change if desired